src/app/features/transactions/components/transaction-dialog/transaction-dialog.component.ts

```diff
@@ -22,6 +22,14 @@ import { ITransactionDialogData } from '../../models/transaction-dialog-data.mod
   styleUrls: ['./transaction-dialog.component.scss']
 })
 export class TransactionDialogComponent {
+  public static DEFAULT_DIALOG_CONFIG = {
+      data: {},
+      width: 'auto',
+      minWidth: 350,
+      hasBackdrop: true,
+      disableClose: true
```

Comment:
The 'disableClose' parameter is the one for making dialog impossible to close when clicking in background